2-mandatory/4-people-i-know.js

```diff
@@ -370,30 +370,41 @@ const friends = [
     ],
   },
 ];
-
 /*
 2) Aged 35 or Older
 In the above object you can see my friends and the colleagues of my friends.
 First, I want you to find all of my friends who are 35 or older.
 */
 
-let thirtyFiveOrOlder = [];
+let thirtyFiveOrOlder = friends.filter((age) => age.age >= 35);
```

Comment:
```suggestion
let thirtyFiveOrOlder = friends.filter((friend) => friend.age >= 35);
```
your code is perfect, the only suggestion would be to change "age" to "friend", because it would be easier to understand "friend.age" instead of "age.age"